frontend/vite.config.mts

```diff
@@ -192,6 +191,9 @@ const ReactCompilerConfig = {
 export default defineConfig({
   // This allows for a dynamic <base> tag in index.html
   base: "./",
+  // experimental: {
+  //   enableNativePlugin: true,
+  // },
```

Comment:
Oh I guess you maybe tried it here. Did this not work? I was using the native resolution in marimo-lsp.